doc/build/changelog/unreleased_XX/README.txt

```diff
@@ -0,0 +1,30 @@
+Individual per-changelog files go here in `.rst` format, which are pulled in by
```

Comment:
instead of `unreleased_XX` i would use `/unreleased_template`

Reply:
I think it's more likely people will find it by seeing README.txt right in the folder that is targeted.

What we *should* be doing here is getting rid of all the "unreleased_XX" folders before _12

Reply:
that also means we need to remove the reference to these folders inside of the changelog_XY.rst files

Reply:
oh and to clarify I'm saying, put this README.txt, with the same thing, in each of unreleased_13 and unreleased_14.     remove unrelased_12 unreleaed_11 unreleased_10.    (yes I added 12 to this, thrre are no 1.2 releases happening and we might need to update our release policy as I'm not really into the "security release" thing once the current vresion has been out for a few months)

Reply:
What if I used a symlink to `../unreleased_template/README.txt` in each folder?

Reply:
from experience windows will not really like the symlink ad git may fail with strange errors, so I'm -1 on that :)

In `unreleased_10` there is still an unreleased changelog

Reply:
any files that are still in those folders move the content to be inline into the corresponding changelog.rst file.

also I'm not too keen on symlinks in git repos, it's just a quick README file it's normal for things to just be copied sometimes.

Reply:
what about this:

* the `unreleased_xx/README.txt` files say "Please read the README.txt in the container folder"
* The .org docs reference the new `/changelog/README.txt' file

The two things I am trying to avoid are:
* having a bunch of README.txt files for each release that are supposed to be the same and can get out of date
* not relying on people to realize "XX" is a placeholder

Reply:
I think we could just place the Readme in the `changelog/` folder without an additional Readme in every folder. I'm sure people can figure it out :)

And we could have actual examples for the names, like `unreleased_14` for 1.4.x releases or `unreleased_15` for 1.5.x.

Reply:
@CaselIT Agreed and now implemented in both PRs.

Reply:
does this README.txt in the changelog/ directory get in the way of sphinx at all, it shouldn't, but you never know w/ sphinx